dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -456,16 +523,22 @@ def test_save_and_load(self):
         test_files = ["csv/guns.csv", "csv/iris.csv"]
 
         def _clean_report(report):
+            global_stats = report["global_stats"]
+            if "correlation_matrix" in global_stats and \
+                    report["global_stats"]["correlation_matrix"] is not None:
+                report["global_stats"]["correlation_matrix"] = np.concatenate(
+                    report["global_stats"]["correlation_matrix"].values).tolist()
```

Comment:
were they not already concatenated? I'll have to inspect this in more detail later.

Reply:
It's a pd.DF, yes? so can we just use the `.tolist()` from pandas?

Reply:
it's a dataframe. So, I have to convert to array, concatenate them and convert to list. df doesn't have .tolist()

Reply:
it'll become list of lists

Reply:
Isn't that what is desired?

Reply:
changed. Initially I thought the assertequal doesn't work with list of lists, so I have to do np.concatenate